File: js/lv1/문자열내p와y의개수.js
function solution(s) {
  const s_lower = s.toLowerCase();

  let num = 0;
  for (let i = 0; i < s.length; i++) {
    if (s_lower[i] === "p") num++;
    if (s_lower[i] === "y") num--;
  }

  const answer = num === 0 ? true : false;

  //   console.log(answer);
  return answer;
}

s1 = "pPoooyY";
answer1 = true;

s2 = "Pyy";
answer2 = false;

solution(s1);
solution(s2);

// function numPY(s){
//       return s.toUpperCase().split("P").length === s.toUpperCase().split("Y").length;
//   }

File: python/lv1/문자열내p와y의개수.py
def solution(s):
    s_lower = s.lower()
    p = s_lower.count('p')
    y = s_lower.count('y')

    print(p == y)
    return p == y

s1 = 'pPoooyY'
answer1 = True

s2 = 'Pyy'
answer2 = False

solution(s1)
solution(s2)